### PR TITLE
Remove includes and excludes

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @ajaybhargavb

File: src/Microsoft.AspNetCore.Diagnostics/project.json [hunk 1]
```diff
@@ -7,17 +7,7 @@
     "nowarn": [
       "CS1591"
     ],
-    "xmlDoc": true,
-    "compile": {
-      "include": "**/*.cs",
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.Diagnostics/project.json]: Doing this causes the symbol packages to have the wrong structure (similar to https://github.com/dotnet/cli/issues/2829).

Comment (pranavkm) [inline src/Microsoft.AspNetCore.Diagnostics/project.json]: That said, it doesn't look like we need either.

**Comment by Eilon** [comment]
:shipit:

**Comment by davidfowl** [comment]
:shipit:

**Pull request merged**